src/main/java/com/emc/mongoose/Main.java

```diff
@@ -87,7 +88,7 @@ public static void main(final String... args)
 					runNode(configWithArgs, extensions, metricsMgr);
 				} else {
 					final Server server = new Server(port);
-					addMetricsService(server);
+					addServices(server, defaultConfig);
```

Comment:
should be `configWithArgs` instead of `defaultConfig`

Reply:
configWithArgs - it is config with applied CLI args != default